### PR TITLE
Allow tables.json to configure shortening //foo:bar to :bar within foo/BUILD

### PR DESCRIPTION
**Pull request by illicitonion**
Also shorten //foo:foo to :foo within foo/BUILD

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

**Comment by illicitonion** [comment]
This was originally in https://github.com/bazelbuild/buildtools/pull/156 but @laurentlb requested it not be enabled by default because it will break many files at Google.

File: build/print_test.go [hunk 1]
```diff
@@ -56,13 +60,20 @@ func TestPrintRewrite(t *testing.T) {
 	for _, in := range ins {
 		prefix := in[:len(in)-len(".in")]
 		out := prefix + ".golden"
+
+		if strings.Contains(out, "/050.") {
```
Comment (laurentlb) [inline build/print_test.go]: Please add a comment (I was initially confused by the "/050.")

**Comment by laurentlb** [comment]
LGTM

**Pull request merged**